docs/loadtest/README.md

```diff
@@ -1,8 +1,7 @@
 # Loadtest
```

Comment:
I missed this during the first review.
You can always feel free to apply suggested changes to occurrences that I missed because they appear elsewhere in the document.

```suggestion
# Load test
```